style.css

```diff
@@ -0,0 +1,761 @@
+  margin-left: 18px;
+  margin-right: 18px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.get-myresume {
+  display: flex;
+  padding: 12px;
+  background: #ff6b00;
+  text-decoration: none;
+  margin-top: 25px;
+  margin-left: 18px;
+  width: 143px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  letter-spacing: 0.03em;
+}
+
+.abt-me-image {
+  align-self: flex-start;
+  width: 30vw;
+  height: 30vh;
+  position: relative;
+  left: 69vw;
+  margin-top: 0;
+}
+
+hr.line-3 {
+  display: flex;
+  margin-top: 5vh;
+  background-color: #e5e5e5;
+  height: 1px;
+  width: 90vw;
+  margin-left: 4vw;
+}
+
+ul.lists {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 12vw;
+  margin-top: 40px;
+  width: 70vw;
+}
+
+.Lan {
+  font-family: "InaiMathi", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: rgb(247, 247, 247);
+}
+
+.Lan::before {
+  content: "\25C6";
+  color: #ff6b00;
+  display: inline-block;
+  width: 1.3em;
+  margin-left: -1em;
+}
+
+.lan-list {
+  display: flex;
+  flex-direction: column;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  align-items: center;
+  text-align: center;
+  color: rgb(247, 240, 240);
+}
+
+.Frame {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: rgb(247, 247, 247);
+}
+
+.Frame::before {
+  content: "\25A0";
+  font-size: 25px;
+  color: #ff6b00;
+  display: inline-block;
+  width: 10px;
+  width: 1em;
+  margin-left: -1em;
+}
+
+.Frame-list {
+  display: flex;
+  flex-direction: column;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  align-items: center;
+  text-align: center;
+  color: rgb(247, 240, 240);
+}
+
+.Sk {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: rgb(247, 247, 247);
+}
+
+.Sk::before {
+  content: "\2022";
+  font-size: 40px;
+  color: #ff6b00;
+  font-weight: bold;
+  display: inline-block;
+  width: 0.6em;
+  margin-left: -1em;
+}
+
+.Sk-list {
+  display: flex;
+  flex-direction: column;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  align-items: center;
+  text-align: center;
+  color: rgb(247, 240, 240);
+}
+
+ul.lists,
+ul.lan-list,
+ul.Frame-list,
+ul.Sk-list {
+  list-style: none;
+}
+
+.forms {
+  display: flex;
+  flex-direction: column;
+}
+
+.upper-form {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+}
```

Comment:
- [OPTIONAL] I would advise you to give some `padding-top` to the form. This will make your design look very identical to that of the Figma template that you chose.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/214639676-366db259-67da-4bb9-9e65-edc6f592fc70.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/214639794-f8da15cf-a6c2-46af-ab7e-b60b8078ad5c.png)